Makefile

```diff
@@ -37,13 +37,46 @@ clobber:	TARGET = clobber
 setup:		TARGET = setup
 test:		TARGET = test
 component-hook:		TARGET = component-hook
+generate-package-kdl:	TARGET = generate-package-kdl
+metadata:		TARGET = metadata
 
 .DEFAULT:	publish
 
-download setup prep build install pre-publish publish validate clean clobber \
-test component-hook unpack patch: $(SUBDIRS)
+download prep build install pre-publish publish validate clean clobber test component-hook generate-package-kdl metadata unpack patch: $(SUBDIRS)
 
 $(SUBDIRS):	FORCE
 	@+echo "$(TARGET) $@" ; $(GMAKE) -s -C $@ $(TARGET)
 
 FORCE:
+
+.PHONY: bootstrap-pkgdev update-pkgdev
+
+# Bootstrap pkgdev before running component setup
+setup: bootstrap-pkgdev $(SUBDIRS)
+
+bootstrap-pkgdev:
+	@echo "Bootstrapping pkgdev via cargo from toasterson/forge"
+	cargo install --git https://github.com/toasterson/forge pkgdev
+
+update-pkgdev:
+	@echo "Updating pkgdev via cargo from toasterson/forge"
+	cargo install --git https://github.com/toasterson/forge pkgdev --force
```

Comment:
Could you please publish forge at crates.io and then we should package it regularly.  This `cargo install` looks like a nasty hack we should avoid.

Thank you.